docs/release-notes/NuGet-6.11.md

```diff
@@ -0,0 +1,62 @@
+---
+title: NuGet 6.11 Release Notes
+description: Release notes for NuGet 6.11 including new features, bug fixes, and DCRs.
+author: mruizmares
+ms.date: 8/1/2024
+ms.topic: conceptual
+---
+
+# NuGet 6.11 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.11**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.11](https://visualstudio.microsoft.com/downloads/) | [<TODO: Full SDK Version>](https://dotnet.microsoft.com/download/dotnet/<TODO: SDKMajorMinorVersionOnly)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
+
+## Summary: What's New in 6.11
+
+* [Feature]: add dotnet nuget why to dotnet CLI - [#11943](https://github.com/NuGet/Home/issues/11943)
+
+### Breaking changes
+
+### Issues fixed in this release
```

Comment:
One of mine is missing as it didn't have a milestone set, which I've done now. Please add https://github.com/NuGet/NuGet.Client/pull/5766

Reply:
Did https://github.com/NuGet/Entropy/tree/main/NuGetReleaseTool/NuGetReleaseTool/AddMilestoneCommand and https://github.com/NuGet/Entropy/tree/main/NuGetReleaseTool/NuGetReleaseTool/GenerateReleaseChangelogCommand catch this issue @martinrrm?
Did you get a change to run those? 

I'd expect that it'd find the issue Donnie is linking

Reply:
Most of the NuGetAuditSuppress issues were in the private repo, so I just created this one for us to include in the release notes:
* https://github.com/NuGet/Home/issues/13679

Reply:
@nkolev92 forgot to run that, let me add the milestone to all of those

Reply:
The command was failing because there are issues with no description, II created a PR to fix that https://github.com/NuGet/Entropy/pull/118